libs/core/src/lib/utils/services/content-density.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+export type ContentDensity = 'cozy' | 'condensed' | 'compact';
+
+@Injectable()
+/**
+ * Service taking care of ContentDensity
+ */
+export class ContentDensityService {
+    contentDensity: BehaviorSubject<ContentDensity>;
+
+    constructor() {
```

Comment:
then you will not need `constructor`, and the only method to add is `subscribe`, and getter `value` for sync access, maybe `next` and `pipe`
you can check this article https://dev.to/alfredoperez/angular-service-to-handle-state-using-behaviorsubject-4818

Reply:
Could you elaborate on the advantage of changing this and adding those methods to this service over what we're doing now?